lib/minify.js

```diff
@@ -125,6 +131,9 @@ function minify(files, options) {
             }
             toplevel = options.parse.toplevel;
         }
+        if (quoted_props) {
+            reserve_quoted_keys(toplevel, quoted_props);
+        }
```

Comment:
I thought about it, and decided to defer until we need it. After all, to hit this we need to be using:
- `--mangle-props`
- `keep_quoted`
- `--timings`

... none of which are commonly used :ghost: